Handling Routes Review/index.js

```diff
@@ -0,0 +1,8 @@
+var http = require("http");
+var routes = require("./routes.js");
```

Comment:
Always use `const` whenever declaring a variable and the value doesn't change.

Reply:
Thanks, @lycoangelo, I updated the var to const.